src/service/cms_api.e

```diff
@@ -36,11 +36,26 @@ feature {NONE} -- Initialize
 			to_implement ("Refactor database setup")
 			if attached setup.storage (error_handler) as l_storage then
 				storage := l_storage
+			elseif attached sqlite_storage as l_sqlite then
```

Comment:
in other case the storage is always CMS_STORAGE_NULL, it's a workaround to make the example works.